client/e2e/pages/authenticated/dashboard/page.spec.ts

```diff
@@ -68,35 +68,38 @@ test.afterEach(async () => {
 
 // 🏷 Annotate test suite as serial
 test.describe.configure({ mode: "serial" });
-// ➰ Loop through the entries of UserRole enum
-for (let [role, value] of Object.entries(UserRole)) {
-  role = "E2E_" + role;
-  const storageState = process.env[role + "_STORAGE"] as string;
-  test.describe(`Test Dashboard for ${value}`, () => {
-    // 👤 run test as this role
-    test.use({ storageState: storageState });
-    test("Test Selfie", async ({ page }) => {
-      // 🛸 Navigate to dashboard page
-      const dashboardPage = new DashboardPOM(page);
+test.describe("Test Dashboard Page", () => {
+  // ➰ Loop through the entries of UserRole enum
```

Comment:
This looks like major diffs in the github UI, but I think the only real change is to wrap everything in a `test.describe("Test Dashboard Page")`? (If there are more changes I'll look more closely)

Reply:
@BCerki correct, attempt to `Redesign looping to improve test describe details`